python/ray/tune/test/automl_searcher_test.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
```

Comment:
Let's make sure this is also run (I think this goes onto travis, see `.travis.yml`)